docs/docs/rasa-pro.mdx

```diff
@@ -70,4 +70,4 @@ Nightly and proactive security patches on your docker image to make sure depende
 
 Modularize your assistant for better scaling and team collaboration.
 
-[Read more here](./concepts/spaces.mdx)
+[Read more here](./operating/spaces.mdx)
```

Comment:
interesting take 🤔

Reply:
I felt having it in the `concepts` section was too prominent given the amount of uncertainty on this topic.